src/framework/theme/styles/_theming.scss

```diff
@@ -10,7 +10,9 @@
  */
 $nb-enable-css-custom-properties: false !default;
```

Comment:
Hi @xxxtonixxx, thanks a lot for contribution! 🙂 
I also see some variables from version 4 here. Could you please remove them as well?
As I can see we only need to define following variables:
```
$nb-enabled-themes: ();
$nb-enable-css-variables: false;
$nb-themes: ();
$nb-themes-non-processed: ();
$nb-themes-export: ();
$theme: null;
$theme-name: null;
```

Reply:
Done! :+1: 

Thanks for your review :smile: